dask/base.py

```diff
@@ -42,6 +42,26 @@
     shorten_traceback,
 )
 
+DistributedClient = None
+get_distributed_client = None
+DISTRIBUTED_AVAILABLE = None
```

Comment:
The only way to set these variables is currently private, so I suggest to make them private as well.
```suggestion
_DistributedClient = None
_get_distributed_client = None
_DISTRIBUTED_AVAILABLE = None
```